### PR TITLE
Changed the version number so its more clear for the users

### PR DESCRIPTION
**Pull request by lawand-dot-io**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/l/last-window-quits.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask "last-window-quits" do
-  version "1.1.3,2024,11"
+  version "1.1.3"
   sha256 "2b6645a0910f27a7021828a44a8744e03db7a7a99b75c3d88ad5fc09c31f29e0"
 
-  url "https://lawand.io/wp-content/uploads/#{version.csv.second}/#{version.csv.third}/last-window-quits-#{version.csv.first}.zip"
+  url "https://lawand.io/wp-content/uploads/2024/11/last-window-quits-#{version}.zip"
```
Comment (krehel) [inline Casks/l/last-window-quits.rb:5]: It is not possible to download the version correctly with this change once the version changes after this month.

Comment (lawand-dot-io) [reply Casks/l/last-window-quits.rb:5]: hmm. but then couldn't I change the cask URL for the next version so that it includes the new month/year?

Comment (ulab) [reply Casks/l/last-window-quits.rb:5]: The version variable is used so you can easily change to the next version without having to change multiple things in the file. 

It allows you to just call `brew bump-cask-pr --version 1.2.3,2025,01 example` for future upgrade for example.

Comment (lawand-dot-io) [reply Casks/l/last-window-quits.rb:5]: yeah, I had a feeling this is the reason. but if this is only to make things easier for myself, I am ok with submitting a pull request like the one I did right now. maybe in the future I would make the URL not include the year/month and make things even simpler

Comment (krehel) [reply Casks/l/last-window-quits.rb:5]: @lawand-dot-io the goal is automation.

Unless you elect to remove the date and year from your URL string at the source and rely on a single version string, the multi-part version will stay.

There are a significant portion of applications that use multi-part versions for this reason.

Comment (lawand-dot-io) [reply Casks/l/last-window-quits.rb:5]: I see. ok this should be fine. thanks. let's close the pull request

Comment (krehel) [reply Casks/l/last-window-quits.rb:5]: Cheers, thanks @lawand-dot-io!

**Pull request closed without merging**